addons/l10n_sk/data/template/account.tax-sk.csv

```diff
@@ -1,31 +1,67 @@
-"id","name","description","invoice_label","amount","amount_type","type_tax_use","sequence","tax_group_id","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@sk","fiscal_position_ids","original_tax_ids"
```

Comment:
Could you maybe add a new column invoice_legal_notes@sk to add the slovak translation ?